create-rust-app/Cargo.toml

```diff
@@ -36,47 +36,47 @@ libsqlite3-sys = { version = "0.25", optional = true, features = ["bundled"] }
 # plugin_auth
 rust-argon2 = { optional=true, version="1.0" }
 rand = { optional=true, version="0.8.5" }
-jsonwebtoken = { optional=true, version="7.2.0" }
-tsync = { optional=true, version="1.3.0" }
-chrono = { optional=true, version = "0.4.22", features = ["serde"] }
+jsonwebtoken = { optional=true, version="8.2.0" }
+tsync = { optional=true, version="1.6.0" }
+chrono = { optional=true, version = "0.4.23", features = ["serde"] }
 
 # plugin_dev
-diesel_migrations = { optional=true, version="2.0.0-rc.1" }
+diesel_migrations = { optional=true, version="2.0.0" }
 
 # plugin_storage
-aws-config = { optional=true, version="0.8.0" }
+aws-config = { optional=true, version="0.14.0" }
 aws-types = { optional=true, version="0.8.0" }
```

Comment:
Updating this to 0.14.0" causes issues, see the details of the failed CI runs here https://github.com/JARAMatch/create-rust-app/actions/runs/3965780378/jobs/6795850402 

I could probably fix this myself, but I can't guarantee I can do so while maintaining current behavior.